<?xml version="1.0"?>
<launch>

    <arg name="receive_topic_name" default="all_cloud"/>
    <arg name="output_topic_name" default="tsuchida_cloud"/>
    <!-- <arg name="frame_id" default="photoneo_center_optical_frame"/> -->
    <arg name="frame_id" default="world"/>
    
    
    
    <node pkg="annotation_package" type="dummy_2_color_cloud" name="dummy_2_color_cloud" output="screen">
        <param name="receive_topic_name" value="$(arg receive_topic_name)"/>
        <param name="output_topic_name" value="$(arg output_topic_name)"/>
        <param name="frame_id" value="$(arg frame_id)"/>    
    </node>
</launch>